model/mesh/wld/z_03_texture_list.go

```diff
@@ -0,0 +1,53 @@
+package wld
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/xackery/encdec"
+	"github.com/xackery/quail/log"
+)
+
+// textureList 0x03 03
+type textureList struct {
+	NameRef      int32
+	TextureNames []string
+}
+
+func (e *WLD) textureListRead(r io.ReadSeeker, fragmentOffset int) error {
+	def := &textureList{}
+	dec := encdec.NewDecoder(r, binary.LittleEndian)
+	def.NameRef = dec.Int32()
+	textureCount := dec.Int32()
+	for i := 0; i < int(textureCount); i++ {
```

Comment:
`textureCount` is going to be off by 1.  

0 for 1 entry, 1 for 2 entries etc.

![image](https://user-images.githubusercontent.com/913141/232925373-b2f5af1b-e667-48e0-8ff3-05d15b62229b.png)

Reply:
Resolved with adding +1 to the iterator